website/docs/r/subnet.html.markdown

```diff
@@ -103,6 +103,17 @@ Class - fvSubnet
 			mac   = "00:1F:20:34:89:AA"
 		}
 	}
+
+	# Disable IP Dataplane Learning for Host(/32) - Under AP -> EPG Subnet
```

Comment:
I see there is a formatting issue in the first example could you format the examples with `terraform fmt` command?

Reply:
I formatted only a new example. I fixed that by formatting all hcl code.